cartridges/int_algolia/cartridge/scripts/algolia/filters/productFilter.js

```diff
@@ -12,6 +41,21 @@ function isInclude(product) {
     // if (product.optionProduct) return false;
     // Do not include bundled product
     if (product.bundled && !(product.priceModel && product.priceModel.price && product.priceModel.price.available)) return false;
+    
+    // Check online status
+    if (!isOnline(product)) return false;
+    
+    // Check searchable status
+    if (!isSearchable(product)) return false;
+    
+    // Check if product has at least one online category
+    if (product.variant && product.getMasterProduct) {
```

Comment:
Can you put a comment to remind that `.getOnlineCategories()` returns nothing on variants? I had forgotten about that

Reply:
Extra reminder added for it.